scripts/sensor/_01_make_evoked.py

```diff
@@ -69,11 +69,10 @@ def run_evoked(*, cfg, subject, session=None):
                                      session=session))
 
         for contrast in cfg.contrasts:
-            cond_1, cond_2 = contrast
-            evoked_diff = mne.combine_evoked([all_evoked[cond_1],
-                                              all_evoked[cond_2]],
-                                             weights=[1, -1])
-            all_evoked[contrast] = evoked_diff
+            conditions = [epochs[x].average() for x in contrast["conditions"]]
```

Comment:
Please don't simply name a list of evokeds "conditions". I don't have a good alternative in mind right now, but `conditions` will lead to confusion.